hibernate-spatial/src/main/java/org/hibernate/spatial/HibernateSpatialConfiguration.java

```diff
@@ -0,0 +1,76 @@
+package org.hibernate.spatial;
+
+import org.geolatte.geom.codec.db.oracle.ConnectionFinder;
+
+import java.io.Serializable;
+
+import org.jboss.logging.Logger;
+
+/**
+ * A global configuration object that is is used by
+ * some Dialects during construction.
+ *
+ * @author Karel Maesen, Geovise BVBA
+ *         creation-date: 8/16/13
+ */
+public class HibernateSpatialConfiguration implements Serializable {
+
+	private static final HSMessageLogger LOG = Logger.getMessageLogger(
```

Comment:
For example, going back to the Logger comment...  This really could just be Logger#getLogger.